### ISSUE TITLE
Update pypi publishing workflow

### ISSUE THREAD

**Issue by reubenmiller** [SoftwareAG/cumulocity-remote-access-local-proxy#166]
The pypy publishing is failing as username/password authentication is no longer supported (we should switch to api tokens).

```
ERROR    HTTPError: 403 Forbidden from https://upload.pypi.org/legacy/          
         Username/Password authentication is no longer supported. Migrate to API
         Tokens or Trusted Publishers instead. See                              
         https://pypi.org/help/#apitoken and                                    
         https://pypi.org/help/#trusted-publishers   
```

**Comment by reubenmiller** [comment]
@switschel Would you be able to update the pypi credentials to use an api token?

**Comment by switschel** [comment]
Fixed with 41262eb59ec2308a47dda77e5231f21315c636fb
